app/assets/v2/js/pages/bounty_details.js

```diff
@@ -167,6 +168,15 @@ var callbacks = {
 
     return [ 'fulfillment_accepted_on', timePeg ];
   },
+  'network': function(key, val, result) {
+    if (val == 'mainnet') {
+      $('#network').addClass('hidden');
+      return [ null, null ];
+    }
+    var warning = 'WARNING: this is a ' + val + ' network bounty, and is NOT real money.  To see mainnet bounties, go to <a href="/explorer">the bounty explorer</a> and search for mainnet bounties.  ';
```

Comment:
Think it'd be worth wrapping these strings with `gettext()` ?

Reply:
🤔 🤷‍♂️

Reply:
256c8e59e2ceaf54a9fee0ea0f1e3e4ae3ed91eb